### ISSUE TITLE
Import de follows

### ISSUE THREAD

**Issue by Lomalarch** [franck-paul/cafe.des-blogueurs.org#1]
Bonjour,
j’ai essayé à plusieurs reprises depuis hier de récupérer les gens que je suis sur mastodon.social. Il me dit que ça s’est bien passé, que les données seront traitées « en temps et heure » et je ne vois rien apparaître. Il y a un truc que j’ai mal fait ?
Bonne journée.

**Comment by franck-paul** [comment]
Nope, visiblement y'a encore des bugs dans ce système, je n'ai réussi à rien non plus hier et j'ai tout fait à la mano ;-)

**Comment by Lomalarch** [comment]
Sweet djizeusse. un bug 0_0

**Comment by franck-paul** [comment]
En fait ça pourrait provenir de compte qui n'existent plus (ou pas atteignable) dans le fichier d'export. Je viens de refaire un test avec un des 2 comptes qui coincent chez moi et j'obtiens un magnifique : **422 Remote account could not be resolved**

**Comment by franck-paul** [comment]
Je viens de laisser [un commentaire](https://github.com/tootsuite/mastodon/commit/540d6efe88717b0da4a23b5069a3caaff5b3241a#commitcomment-21714517) sur le commit censé régler ça, on verra bien.
